imports/plugins/included/jobcontrol/server/jobs/cart.js

```diff
@@ -60,6 +60,7 @@ export function cartCleanupJob() {
             _id: cart.userId,
             emails: []
           });
+          Hooks.Events.run("afterAccountsRemove", cart.userId, user._id);
```

Comment:
The first arg should be `null` because this is a background job, which does not run with a user context.